images/capi/packer/ami/packer.json

```diff
@@ -14,7 +14,7 @@
     "containerd_version": null,
     "containerd_sha256": null,
     "containerd_url": "https://storage.googleapis.com/cri-containerd-release/cri-containerd-{{user `containerd_version`}}.linux-amd64.tar.gz",
-    "containerd_cri_socket": "/run/docker/libcontainerd/docker-containerd.sock",
+    "containerd_cri_socket": "/var/run/containerd/containerd.sock",
```

Comment:
I saw his comment, but on flatcar the `containerd.service` is listening on `/var/run/docker/libcontainerd/docker-containerd.sock` right?

Reply:
Right.

The Flatcar-specific configs are actually defined in [`packer/ami/flatcar.json`](https://github.com/kubernetes-sigs/image-builder/blob/da65516dbe9091f49f52950b9a68e1ecff8c228c/images/capi/packer/ami/flatcar.json#L7), which has already correctly `/run/docker/libcontainerd/docker-containerd.sock`.
On the other hand, [`packer/ami/packer.json`](https://github.com/kubernetes-sigs/image-builder/blob/da65516dbe9091f49f52950b9a68e1ecff8c228c/images/capi/packer/ami/packer.json#L17) is supposed to have the default configs for non-Flatcar distros.
Previously we had a wrong value `/run/docker/libcontainerd/docker-containerd.sock` in the default config file.
In this PR we are fixing it to `/var/run/containerd/containerd.sock`, as Moshe pointed out.